lib/yardstick/measurement.rb

```diff
@@ -34,7 +34,7 @@ def initialize(rule)
     #
     # @api public
     def ok?
-      @result == true || skip?
+      @result.equal?(true) || skip?
```

Comment:
The `@result` ivar is set to either `true`, `false` or `:skip`. The `#ok?` method returns true if the result is true or skipped.